invoices_app/tests.py

```diff
@@ -130,6 +136,53 @@ def test_supplier_invoices_list_only_taxpayer_invoices(self):
             [taxpayer.id for taxpayer in response.context['object_list']]
         )
 
+    def test_supplier_with_2invoices_list_only_two_invoices(self):
+        self.client.force_login(self.user2)
+
+        other_invoice_data = self.invoice_creation_valid_data
+        other_invoice_data['taxpayer'] = self.taxpayer1_user2
+        other_invoice_data['user'] = self.user2
+        invoice1 = InvoiceArg.objects.create(**other_invoice_data)
```

Comment:
Are you familiar with the Factory concept? We should discuss in order to save time on tests